math/CHANGELOG.md

```diff
@@ -39,7 +39,9 @@ Ref: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.j
 ### Bug Fixes
 
 * [#16266](https://github.com/cosmos/cosmos-sdk/pull/16266) fix: legacy dec power mut zero exponent precision.
+* [#17352](https://github.com/cosmos/cosmos-sdk/pull/17352) Ensure that modifying the argument to `NewIntFromBigInt` doesn't mutate the returned value.
 * [#18211](https://github.com/cosmos/cosmos-sdk/pull/18211) RelativePow now returns 1 when 0^0, before it was returning the scale factor.
+* [#18214](https://github.com/cosmos/cosmos-sdk/pull/18214) Ensure that modifying the argument to 
```

Comment:
End of the sentence is missing.
Do we need this `release/v0.50.x`?

Reply:
We backported the same bug for v0.47 so I guess we want this here too? I'll fix the comment of the backported bugs